services/cd-service/pkg/service/release_train_prognosis.go

```diff
@@ -48,7 +48,7 @@ func (s *ReleaseTrainPrognosisServer) GetReleaseTrainPrognosis(ctx context.Conte
 	}
 	dbHandler := t.Repo.State().DBHandler
 	var prognosis rp.ReleaseTrainPrognosis
-	_ = dbHandler.WithTransaction(ctx, true, func(ctx context.Context, transaction *sql.Tx) error {
+	_ = dbHandler.WithTransaction(ctx, false, func(ctx context.Context, transaction *sql.Tx) error {
```

Comment:
This deserves a comment
```suggestion

        const readOnly=false // generally speaking, the prognoses only read data, however `DBSelectCommitIdAppReleaseVersions` requires write access for a temporary table.
	_ = dbHandler.WithTransaction(ctx, readOnly, func(ctx context.Context, transaction *sql.Tx) error {
```